capstone_project/src/main/webapp/scripts/maps_script.js

```diff
@@ -30,8 +30,7 @@ function createMap() { // eslint-disable-line no-unused-vars
  */
 function loadMarkersOntoMap(map) {
   fetch('/school-data').then((response) => response.json()).then((schools) => {
-    for (let i = 0; i < schools.length; i++) {
-      const school = schools[i];
+    for (school of schools) {
```

Comment:
I think not using let or const declares school as a global variable. I think const would be used here since I don't think you modify the school variable

Reply:
Oh that makes sense, it's fixed now! @Leoism